llms/mlx_lm/utils.py

```diff
@@ -71,7 +71,7 @@ def get_model_path(path_or_hf_repo: str) -> Path:
 
 
 def generate_step(
-    prompt: mx.array, model: nn.Module, temp: float = 0.0
+    prompt: mx.array, model: nn.Module, temp: float = 0.0, colorize: bool = False
```

Comment:
Can you change the name of this parameter to `return_probability`? It might be useful for other things and it's more clear what it does in that case.

Reply:
Should we change the --colorize command line argument too? 
I like the idea of giving more control over the output in future, if this is used 🤷‍♂️. 
People may like to change thresholds or colors (being colorblind this makes sense to me).

Reply:
I think it's ok the way you have it. That command line really does colorize so it's aptly named. If we eventually make the output more flexible via a callback and want to expose more option to the CLI we can change it then.